.github/workflows/ci.yml

```diff
@@ -59,10 +59,7 @@ jobs:
           build-args: |
             VERSION=latest
             GIT_COMMIT=${GITHUB_SHA}
-      - name: Scan docker image
-        uses: azure/container-scan@v0.1
+      - name: Run Trivy vulnerability scanner
+        uses: hypertrace/github-actions/trivy-image-scan@main
         with:
-          image-name: hypertrace/hypertrace-collector:latest
-        env:
-          DOCKLE_HOST: "unix:///var/run/docker.sock"
-        continue-on-error: true
+          image: hypertrace/hypertrace-collector
```

Comment:
this uses gradle to try to calculate the appropriate tag; if that's not right here (looks like build above is custom), then it needs to be passed in via `tag` arg:
https://github.com/hypertrace/github-actions/blob/06583dbad04268ad71b030c70beb1500e260b540/trivy-image-scan/action.yml#L7-L9

Reply:
Done.